Source/Scene/GlobeDepth.js

```diff
@@ -0,0 +1,235 @@
+/*global define*/
+define([
+        '../Core/Color',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/PixelFormat',
+        '../Renderer/ClearCommand',
+        '../Renderer/PixelDatatype',
+        '../Shaders/PostProcessFilters/PassThrough'
+    ], function(
+        Color,
+        defined,
+        destroyObject,
+        PixelFormat,
+        ClearCommand,
+        PixelDatatype,
+        PassThrough) {
+    "use strict";
+    /*global WebGLRenderingContext*/
+
+    /**
+     * @private
+     */
+    var GlobeDepth = function(context) {
+        this._colorTexture = undefined;
+        this._depthStencilTexture = undefined;
+        this._depthStencilGlobeTest = undefined;
+
+        this.framebuffer = undefined;
+        this._copyDepthFramebuffer = undefined;
+
+        this._clearColorCommand = undefined;
+        this._copyColorCommand = undefined;
+        this._copyDepthCommand = undefined;
+
+        var supported = updateFramebuffers(this, context);
+        if (supported) {
+            updateCopyCommands(this, context);
+        }
+
+        this.supported = supported;
```

Comment:
Should this be a property getter or at least documented `@readonly` for internal documentation?